clang/test/CXX/drs/dr13xx.cpp

```diff
@@ -423,13 +423,13 @@ namespace dr1359 { // dr1359: 3.5
 #if __cplusplus >= 201103L
   union A { constexpr A() = default; };
   union B { constexpr B() = default; int a; }; // #dr1359-B
-  // cxx11-17-error@-1 {{defaulted definition of default constructor is not constexpr}}
+  // cxx11-17-error@-1 {{defaulted definition of default constructor that marked constexpr but never produces a constant expression is a C++23 extension}}
```

Comment:
11 through 17 we expect a 23 extension warning, but what happens in 20 mode?

Reply:
Same, In C++20 mode such constructor is considered as a valid and producing constexpr due to https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1331r2.pdf , so no extension warning there.